lib/src/modules/meta/meta.scss

```diff
@@ -0,0 +1,3 @@
+.meta-text {
+  @apply truncate;
```

Comment:
This is applied like this, so we can overwrite styles to allowing wrapping when required. `!important` and tailwind makes this tricky.

Reply:
💚 A sass file just for this is a little sad. Either add a prop that controls this classname within the component, or move everything over to here

Reply:
Let me see if I can still use `truncate` and when I pass `whitespace` whatever, it fixed the issue.